pkg/restapi/operation/operations.go

```diff
@@ -210,18 +244,32 @@ func (c *Operation) createDocumentHandler(rw http.ResponseWriter, req *http.Requ
 		return
 	}
 
+	dbVaultName, err := c.vaultCollection.getCouchDBVaultNameFromVaultID(vaultID)
```

Comment:
I'm a bit confused about the usages of "Vault name" and "Vault ID" here. Is "vault name" the reference ID?

Reply:
done!